docs/docs/destinations/README.md

```diff
@@ -23,16 +23,16 @@ The docs below discuss features common to all Destinations. See the [docs for a
 
 ### Using destinations in workflows
 
-You can send data to Destinations in [Node.js code steps](/workflows/steps/code/), too, using `$send` functions.
+You can send data to Destinations in [Node.js code steps](/workflows/steps/code/), too, using `$.send` functions.
 
-`$send` is an object provided by Pipedream that exposes destination-specific functions like `$send.http()`, `$send.s3()`, and more. **This allows you to send data to destinations programmatically, if you need more control than the default actions provide**.
+`$.send` is an object provided by Pipedream that exposes destination-specific functions like `$.send.http()`, `$.send.s3()`, and more. **This allows you to send data to destinations programmatically, if you need more control than the default actions provide**.
 
-Let's use `$send.http()` to send an HTTP POST request like we did in the Action example above. [Add a new Action](/components/actions/#adding-a-new-action), then search for "**Run Node.js code**":
+Let's use `$.send.http()` to send an HTTP POST request like we did in the Action example above. [Add a new Action](/components/actions/#adding-a-new-action), then search for "**Run Node.js code**":
 
 [Create an endpoint URL](https://requestbin.com), adding the code below to your code step, with the URL you created:
 
 ```javascript
-$send.http({
+$.send.http({
```

Comment:
These examples won't work as written, since you only have access to the `$` within the `run` method of Node.js components. So you'll just need to wrap with the standard `defineComponent` code.

Reply:
I didn't comment in the other docs, but we'll need to modify the code there, too.

Reply:
Ah I see, I thought the `defineComponent` line was implict and wasn't editable like the old `run` function in the past workflow examples. Sure I'll update all to match.

Reply:
Probably should make a note in the Components reference to make sure developers understand they are defining a component and the `defineComponent` function isn't necessary.